contents/docs/experiments/best-practices.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: Experiment best practices
+sidebar: Docs
+showTitle: true
+---
+
+## 1. Only include users affected by the change
+
+Including users who aren't affected by the change being tested can lead to unreliable results. For example, if you're testing a new onboarding flow, you don't want to include users who have already completed the flow.
```

Comment:
I wonder if "experiment" would be a better word here instead of "the change being tested"?

On the first time reading the sentence, "the change being tested" is a little hard to parse.